packages/kestrel_core/src/kestrel/mapping/data_model.py

```diff
@@ -298,18 +304,35 @@ def translate_projection_to_native(
 
 
 @typechecked
-def translate_projection_to_ocsf(
-    dmm: dict,
+def translate_projection_entity_to_ocsf(
```

Comment:
This name seems a bit weird; maybe `translate_entity_projection_to_ocsf` would read better?

Reply:
Good point!